src/sentry/sentry_apps/external_issues/issue_link_creator.py

```diff
@@ -33,7 +32,7 @@ def run(self) -> PlatformExternalIssue:
 
     def _verify_action(self) -> None:
         if self.action not in VALID_ACTIONS:
-            raise SentryAppSentryError(APIUnauthorized(f"Invalid action '{self.action}'"))
+            raise SentryAppSentryError(message=f"Invalid action: {self.action}")
```

Comment:
`APIUnauthorized` has status code 401

```suggestion
            raise SentryAppSentryError(message=f"Invalid action: {self.action}", status_code=401)
```